server/src/main/java/dev/slimevr/tracking/processor/HumanPoseManager.java

```diff
@@ -144,7 +144,7 @@ private void initializeComputedHumanPoseTracker() {
 					VRServer.getNextLocalTrackerId(),
 					"human://CHEST",
 					"Computed chest",
-					TrackerPosition.CHEST,
+					TrackerPosition.UPPER_CHEST,
```

Comment:
is this fine?

Reply:
It really doesn't matter if it's CHEST or UPPER_CHEST as long as it's changed in the rest of the code (doesn't affect config or solarxr I mean). 
So yes it's fine, I did change it in other parts of the code already.